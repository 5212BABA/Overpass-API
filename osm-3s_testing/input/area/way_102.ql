way(102);node(area)(if:id()>=10200 && id()<=10299);out;
