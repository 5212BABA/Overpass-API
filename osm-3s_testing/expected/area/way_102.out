<?xml version="1.0" encoding="UTF-8"?>
<osm version="0.6" generator="Overpass API">
<note>The data included in this document is from www.openstreetmap.org. The data is made available under ODbL.</note>
<meta osm_base=""/>

  <node id="10201" lat="51.2500001" lon="7.1510001"/>
  <node id="10202" lat="51.2510001" lon="7.1520001"/>
  <node id="10203" lat="51.2520001" lon="7.1510001"/>
  <node id="10204" lat="51.2510001" lon="7.1500001"/>
  <node id="10214" lat="51.2500002" lon="7.1510002"/>
  <node id="10215" lat="51.2500002" lon="7.1510001"/>
  <node id="10216" lat="51.2500002" lon="7.1510000"/>
  <node id="10226" lat="51.2510002" lon="7.1520000"/>
  <node id="10227" lat="51.2510001" lon="7.1520000"/>
  <node id="10228" lat="51.2510000" lon="7.1520000"/>
  <node id="10231" lat="51.2520000" lon="7.1510001"/>
  <node id="10232" lat="51.2520000" lon="7.1510002"/>
  <node id="10238" lat="51.2520000" lon="7.1510000"/>
  <node id="10242" lat="51.2510000" lon="7.1500002"/>
  <node id="10243" lat="51.2510001" lon="7.1500002"/>
  <node id="10244" lat="51.2510002" lon="7.1500002"/>
  <node id="10254" lat="51.2505002" lon="7.1515002"/>
  <node id="10255" lat="51.2505002" lon="7.1515001"/>
  <node id="10256" lat="51.2505002" lon="7.1515000"/>
  <node id="10257" lat="51.2505001" lon="7.1515000"/>
  <node id="10258" lat="51.2505000" lon="7.1515000"/>
  <node id="10261" lat="51.2515000" lon="7.1515001"/>
  <node id="10262" lat="51.2515000" lon="7.1515002"/>
  <node id="10266" lat="51.2515002" lon="7.1515000"/>
  <node id="10267" lat="51.2515001" lon="7.1515000"/>
  <node id="10268" lat="51.2515000" lon="7.1515000"/>
  <node id="10271" lat="51.2515000" lon="7.1505001"/>
  <node id="10272" lat="51.2515000" lon="7.1505002"/>
  <node id="10273" lat="51.2515001" lon="7.1505002"/>
  <node id="10274" lat="51.2515002" lon="7.1505002"/>
  <node id="10278" lat="51.2515000" lon="7.1505000"/>
  <node id="10282" lat="51.2505000" lon="7.1505002"/>
  <node id="10283" lat="51.2505001" lon="7.1505002"/>
  <node id="10284" lat="51.2505002" lon="7.1505002"/>
  <node id="10285" lat="51.2505002" lon="7.1505001"/>
  <node id="10286" lat="51.2505002" lon="7.1505000"/>

</osm>
